src/main/java/frc/robot/subsystems/ShooterHoodSubsystem.java

```diff
@@ -91,19 +98,24 @@ public void stowHood() {
   }
 
   public double getHoodPos() {
-    return hoodPos;
+    return hoodEncoder.getPosition();
```

Comment:
Just out of curiosity, why did this move here?  It used to be in periodic().  Unless we truly need to get the hood position more frequently than regular period, it's better to read it once and cache the result.

Reply:
Fixed